helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: kyverno.all.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: webhooks
+    rules:
+    - alert: KyvernoWebhooksPresent
```

Comment:
```suggestion
    - alert: KyvernoWebhookHasNoAvailableReplicas
```

Reply:
I also wonder if this alert could be made generic to any webhook. Might require quite some promql magic though